### PR TITLE
registry repository count function bug fix

### PR DESCRIPTION
**Pull request by ndufreche**
Little correction on a old copy past fail.

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
please, next time use a fork.

I relaunch the build.

File: Bundle/Registry/Repository.php [hunk 1]
```diff
@@ -146,8 +146,8 @@ public function findEntity($id)
 
     public function count($descriminator = null)
     {
-        if (is_null($identifier)) {
-            $identifier = $this->getEntityName();
+        if (is_null($descriminator)) {
```
Comment (eric-chau) [inline Bundle/Registry/Repository.php]: Use ``null === $descriminator`` instead of using a function =)

**Comment by eric-chau** [comment]
Agree with @mickaelandrieu. Nice catch @ndufreche, thank you.

**Comment by ndufreche** [comment]
I have recreate the PR on my fork

**Pull request closed without merging**